### ISSUE TITLE
Add support for disk partitioning and xfs creation

### ISSUE THREAD

**Issue by kdunn-pivotal** [andreasscherbaum/gpdb-ansible#3]
Refer to our conversation: https://pivotal.slack.com/archives/C4WUTFL30/p1504291458000484

**Comment by andreasscherbaum** [comment]
There is actually a Todo somewhere as a comment in my Playbook - but I'm not sure that the GPDB Playbook is the right place. This is something which needs to be done when you configure the host, or even earlier: when you setup the host. Once the partitions are created and the OS is installed, it's too late to change that.

https://github.com/andreasscherbaum/gpdb-ansible/blob/master/roles/gpdb4/tasks/redhat.yml#L184

**Comment by kdunn-pivotal** [comment]
Fair enough - I'm not too sure how your demarcations are drawn with this playbook, but to be clear, this is disk partitioning + xfs formatting for GPDB data disks/dirs only, not the OS itself.

**Comment by andreasscherbaum** [comment]
Is your use case that you have a partition for the OS, and another one for the database?
